### PR TITLE
update readme with new formulation name

### PR DESCRIPTION
**Pull request by jalving**

### CODE REVIEW

**Review by rmisener — CHANGES_REQUESTED** [review]
Thanks for remembering to update the README. In addition to what you have already done, would you consider changing the introductory paragraph from: "(such as neural networks)" to "(neural networks and gradient-boosted trees)" and consider changing "import sequential Keras models" to "import sequential Keras and ONNX models"?

**Review by rmisener — APPROVED** [review]
